src/status_im/ui/components/colors.cljs

```diff
@@ -16,6 +16,9 @@
 (def gray-notifications "#4A5054cc") ;; Used for notifications background
 (def gray-border "#ececf0")
 (def blue "#4360df") ;; Used as main wallet color, and ios home add button
+(def blue-transparent "rgba(67, 96, 223, 0.10)")
+(def blue-transparent-20 "rgba(67, 96, 223, 0.2)")
```

Comment:
Do we really need two slightly different transparent blue colors?

Reply:
@goranjovic actually, not. Will remove second which is not used.